crates/optimism/payload/src/payload.rs

```diff
@@ -37,6 +37,8 @@ pub struct OptimismPayloadBuilderAttributes {
     pub transactions: Vec<WithEncoded<TransactionSigned>>,
     /// The gas limit for the generated payload
     pub gas_limit: Option<u64>,
+    /// EIP-1559 parameters for the generated payload
+    pub eip_1559_params: Option<B64>,
```

Comment:
This needs to be included within the computation of the payload ID, in `optimism_payload_id`.

Reply:
Added
https://github.com/paradigmxyz/reth/pull/11473/files#diff-36ba15633729e1f0ec511982b2323edfd67d706821dcd25b3caa2aada7c4b7d8R319-R322